### DISCUSSION TITLE
Why can't I @mention someone who has editing rights?

### DISCUSSION THREAD

**Discussion by EduNetEurope** [outline/outline#6900] [General]
It looks like users can only be @mentioned when the collection is shared with them but not when they have been invited / added to a set of nested documents within a collection?

I am in a document.
I click on "Share". User A is listed with editing rights. 
When I add a comment and try to @mention user A, user A cannot be selected.

**Comment by tommoor** [comment]
Sounds like a bug, filed here: https://github.com/outline/outline/issues/6901
